brands/shop/clothes.json

```diff
@@ -1572,6 +1572,15 @@
       "shop": "clothes"
     }
   },
+  "shop/clothes|Jockey": {
+    "tags": {
+      "brand": "Jockey International",
```

Comment:
I would change `brand` and `name` here to just say "Jockey".  These are more for the name that that they use on the storefronts, not really the official name of the company.